src/views/Home.vue

```diff
@@ -13,7 +13,22 @@
       <div class="rf-col-8">
 
         <IntroSection></IntroSection>
-        <SearchBar></SearchBar>
+
+        <SearchBar>
+             <form>
+              <div class="rf-search-bar" id="search-input">
+                  <label class="rf-label" for="search-input-input">Label de la barre de recherche</label>
+                    <input type="search" class="rf-input" id="search-input-input" name="search-input-input" v-model="recherche" @keydown.enter.prevent="goToResearch(recherche)" placeholder="Rechercher un mot clé, une expression, une référence..." aria-describedby="basic-addon2">
```

Comment:
- Utilisation du caractère typographique « … » au lieu de « ... ».
 - Suppression de `aria-describedby` inutile car le champ est déjà décrit par un label.
 - Suppression du binding sur `keydown`.

```suggestion
                    <input type="search" class="rf-input" id="search-input-input" name="search-input-input" v-model="recherche" placeholder="Rechercher un mot clé, une expression, une référence…">
```